app/main.rb

```diff
@@ -0,0 +1,145 @@
+require_relative './book_methods'
+require_relative './author_list'
+require_relative './game_list'
+require_relative './movie'
+require_relative '../file/read_write'
+require_relative './source'
+require_relative './label_methods'
+require_relative './music_album'
+require_relative './genre'
+
+class Main
+  def initialize(options)
+    @options = options
+  end
+
+  include AuthorList
+  include GameList
+
+  def print_menu
+    puts "\n Welcome to Catelogy of my things APP"
+    @options.each { |option| puts "#{option[:order]}.#{option[:message]}" }
+  end
+
+  def user_input
+    gets.chomp.to_i
+  end
+
+  def prompt_create_movie
+    puts('Enter date of publication (YYYY-MM-DD):')
+    publish_date = gets.chomp.to_s
+    puts('Is the Movie is silent Y/N:')
+    silent = gets.chomp.to_s.downcase == 'y'
+    puts('Enter Source Name:')
+    source_name = gets.chomp.to_s
+    movie = Movie.new(silent, publish_date, false)
+    source = Source.new(source_name)
+    ReadWrite.new(Movie).add_file({ type: movie.class, silent: silent, publish_date: publish_date, archived: false })
+    ReadWrite.new(Source).add_file({ type: source.class, name: source_name, items: [] })
+    puts('Movie successfully added')
+  end
+
+  def parse_bool
+    option = gets.chomp
+    case option.to_s.upcase
+    when 'Y'
+      option = true
+    when 'N'
+      option = false
+    end
+    option
+  end
+
+  # rubocop:disable Metrics/MethodLength
+  # rubocop:disable Metrics/CyclomaticComplexity
+  def promot_user
+    @books = BookApp.new
+    @books.read_books
+    @labels = LabelApp.new
+    @labels.read_labels
+    loop do
+      print_menu
+      input = user_input
+      case input
+      when 1 then @books.list_all_books
+      when 2 then list_all(MusicAlbum)
+      when 3 then list_all(Movie)
+      when 4 then list_games
+      when 5 then list_all(Genre)
+      when 6 then @labels.list_all_labels
+      when 7 then list_author
+      when 8 then @books.create_book
+      when 9 then list_all(Source)
+      when 10 then create_music_album
+      when 11 then prompt_create_movie
+      when 12 then @labels.create_label
+      when 13 then add_game
+      else
+        @books.save_books
+        @labels.save_labels
+        break
+      end
+      break unless input.positive? && input <= 13
+    end
+  end
+  # rubocop:enable Metrics/MethodLength
+
+  # rubocop:enable Metrics/CyclomaticComplexity
+  def list_all(type)
+    data = ReadWrite.new(type).read_all_records
+    if data.empty?
+      puts("No #{type} found. \n")
+    else
+      puts("List of #{type}: \n")
+      format_data(data)
+    end
+  end
```

Comment:
[OPTIONAL] Your app is working correctly, great job! However, when you display the data, sometimes you leave blank lines, and sometimes you don't, it would be easier to read if you left some blank spaces between lines:

<img width="318" alt="Screen Shot 2022-08-12 at 6 14 30 PM" src="https://user-images.githubusercontent.com/83717285/184457155-1356594a-885a-4f0a-a3d1-48b0ef64cc0f.png">